prometheus.yml.tmpl

```diff
@@ -7,6 +7,13 @@ rule_files:
   - /rules.yml
   - /etc/prometheus/rules-l1.yml
   - /etc/prometheus/rules-ln.yml
+  - /etc/prometheus/alert_rules-ln.yml
```

Comment:
@KarineValenca, the alerting rules should differ based on the bb-promster level. This line will probably need to be changed to make it happend.